pages/bills/[id]/index.js

```diff
@@ -12,14 +12,16 @@ export default function BillPageDetails() {
   if (isLoading || error) return <h2>Loading Bill..</h2>;
 
   return (
-    <BillDetails title={bill.title} total={bill.amount}>
+    <BillDetails title={bill.title} amount={bill.amount}>
       {bill.participants.map((participant, index) => (
         <li key={index}>
           {participant.name} paid {participant.paid} $, and{" "}
           {participant.balance}.
         </li>
       ))}
-
+      <Link href={`/bills/${id}/edit`}>
+        <button type='button'>Edit Bill</button>
+      </Link>
```

Comment:
buttons are not valid descendants of anchors, to have a link looking like a button requires to style the link.